app/scripts/modules/cloudfoundry/src/pipeline/stages/deployService/cloudfoundryDeployServiceStage.module.ts

```diff
@@ -52,30 +53,20 @@ PipelineConfigValidator.registerValidator(
 );
 
 PipelineConfigValidator.registerValidator(
-  'validDeployServiceParameterJson',
+  'validDeployServiceParameterJsonOrYaml',
   new class implements IStageOrTriggerValidator {
-    private validationMessage(validationConfig: IServiceFieldValidatorConfig): string {
-      return (
-        validationConfig.message ||
-        `<strong>${this.printableFieldLabel(validationConfig)}</strong> should be a valid JSON string.`
-      );
-    }
-
-    private printableFieldLabel(config: IServiceFieldValidatorConfig): string {
-      return upperFirst(config.fieldLabel || config.fieldName);
-    }
-
-    private fieldIsValid(stage: IStage | ITrigger, config: IServiceFieldValidatorConfig): boolean {
-      const serviceInput = get(stage, 'manifest');
-      const manifestSource: any = get(serviceInput, sourceStruct(serviceInput));
-      const content: any = get(manifestSource, config.fieldName);
-
-      if (!content) {
+    private isJson(value: string): boolean {
+      try {
+        JSON.parse(value);
         return true;
+      } catch (e) {
+        return false;
       }
+    }
 
+    private isYaml(value: string): boolean {
       try {
-        JSON.parse(content);
+        safeLoad(value, null);
```

Comment:
nit picky - you don't need to specify a second parameter
`safeLoad(value);` 
would work fine